NeosModLoader/NeosModLoaderHeadless.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectGuid>{4a54bcdf-0386-43fc-9aa5-d49ce7d33dd8}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>NeosModLoader</RootNamespace>
+    <AssemblyTitle>NeosModLoader</AssemblyTitle>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo> 
+    <TargetFramework>net462</TargetFramework>
+    <FileAlignment>512</FileAlignment>
+    <Deterministic>true</Deterministic>
+    <NeosPath>$(MSBuildThisFileDirectory)NeosVR</NeosPath>
+    <NeosPath Condition="Exists('C:\Program Files (x86)\Steam\steamapps\common\NeosVR\')">C:\Program Files (x86)\Steam\steamapps\common\NeosVR\</NeosPath>
+    <NeosPath Condition="Exists('$(HOME)/.steam/steam/steamapps/common/NeosVR/')">$(HOME)/.steam/steam/steamapps/common/NeosVR/</NeosPath>
+    <CopyToPlugins Condition="'$(CopyToPlugins)'==''">true</CopyToPlugins>
+    <DebugSymbols Condition="'$(Configuration)'=='Release'">false</DebugSymbols>
+    <DebugType Condition="'$(Configuration)'=='Release'">None</DebugType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Lib.Harmony" Version="2.2.0" />
+    <Reference Include="BaseX">
+      <HintPath>$(NeosPath)HeadlessClient\BaseX.dll</HintPath>
+    </Reference>
+    <Reference Include="FrooxEngine">
+      <HintPath>$(NeosPath)HeadlessClient\FrooxEngine.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>$(NeosPath)HeadlessClient\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent" Condition="'$(CopyToPlugins)'=='true'">
+      <Copy SourceFiles="$(TargetPath)" DestinationFolder="$(NeosPath)HeadlessClient\Libraries" />
```

Comment:
I am like 90% certain that this would break the Github actions CI system, since it doesn't have assemblies for the headless. I don't think there is any good way for us to have reference assemblies of the headless client sadly, so it'd need to be excluded from the CI.